packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/backups.controller.js

```diff
@@ -44,6 +44,22 @@ export default class PciProjectStorageDatabaseBackupsCtrl {
     );
   }
 
+  getRegions(backup) {
+    let regionString = '-';
+    if (backup.regions.length > 0) {
+      regionString = backup.regions
+        .map((region) =>
+          this.ovhManagerRegionService.getTranslatedMacroRegion(region.name),
+        )
+        .join(', ');
+    } else if (backup.region) {
+      regionString = this.ovhManagerRegionService.getTranslatedMacroRegion(
+        backup.region,
+      );
+    }
+    return regionString;
```

Comment:
i guess we could complete it like this to be more accurate :
```suggestion
return (backup.regions || [{name: backup.region}])
        .map((region) =>
          this.ovhManagerRegionService.getTranslatedMacroRegion(region.name),
        )
        .join(', ');
```